eng/Packages.props

```diff
@@ -28,7 +28,10 @@
     <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
     <PackageVersion Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindowsVersion)" />
     <PackageVersion Include="System.Text.Encoding.CodePages" Version="$(SystemTextEncodingCodePagesVersion)" />
-    <PackageVersion Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
+    <!-- Ensure the latest STJ version for .NET-SDK delivered components -->
+    <PackageVersion Include="System.Text.Json" Version="$(SystemTextJsonVersion)" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
+    <!-- Avoid bumping to STJ 8.0.4 for VS-delivered bits until VS updates later. This version was what was referenced in MSBuild 17.8.5 -->
```

Comment:
I don't know if this still holds true too. @rainersigwald, can we just use the 8.0.4 of STJ for VS too?

Reply:
not for 17.8 AFAIK.